### PR TITLE
Calculating MA21 for per million columns

### PR DESCRIPTION
**Pull request by talkgarima**

### CODE REVIEW

File: backend/metrics/calculations.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def calculate_hospitalizations(
     return df
 
 
-def _moving_average_grouped(df: pd.DataFrame, group_columns: list[str], target_column: str, window_size) -> pd.Series:
```
Comment (Nozziel) [inline backend/metrics/calculations.py:49]: please keep this. upgrade to python 3.8 or 3.9...
Or if you ahve to switch to captital L List and import it from typing
from typing import List

File: backend/metrics/calculations.py [hunk 2]
```diff
@@ -103,7 +103,7 @@ def fourteen_day_avg_ratio(values: pd.Series) -> pd.Series:
 def extend_and_impute_metrics(
         raw_metrics: pd.DataFrame,
         hospitalizations: pd.DataFrame,
-        grouping_columns: list[str]
```
Comment (Nozziel) [inline backend/metrics/calculations.py:106]: restore

File: backend/metrics/calculations.py [hunk 3]
```diff
@@ -157,7 +157,14 @@ def extend_and_impute_metrics(
...
     # moving averages of some of our calculated columns
-    for column in ['delta.positivity', 'delta.hospitalized', 'delta.active']:
+    for column in ['delta.positivity', 'delta.hospitalized', 'delta.active'
+                   , 'delta.confirmed.ratio_per_million', 'delta.deceased.ratio_per_million',
+                   'total.confirmed.ratio_per_million', 'total.deceased.ratio_per_million']:
         df.loc[:, f'MA.21.{column}'] = _moving_average_grouped(df, grouping_columns, column, rolling_window)
 
+    # moving averages of some of our calculated columns
+    # for column in ['delta.positivity', 'delta.hospitalized', 'delta.active']:
+    #     df.loc[:, f'MA.21.{column}'] = _moving_average_grouped(df, grouping_columns, column, rolling_window)
```
Comment (Nozziel) [inline backend/metrics/calculations.py:167]: remove commented out code

File: tests/integration_tests/test_update_dashboard_task.py [hunk 4]
```diff
@@ -49,7 +49,7 @@ def mock_get_dataframe(self, storage_name):
             if storage_name == UpdateDashboardTask.storage_hospitalizations:
                 return pd.read_csv(
                     os.path.join(THIS_DIR, '../samples/Dashboard PDF Development - hospitalization.csv'))
-            raise ValueError(f'Did not expect {storage_name=}')
```
Comment (Nozziel) [inline tests/integration_tests/test_update_dashboard_task.py:52]: again an issue with the old python version you are using ...

**Review by Nozziel — CHANGES_REQUESTED** [review]

File: tests/integration_tests/test_update_dashboard_task.py (hunk 4 above)
Comment (talkgarima) [reply tests/integration_tests/test_update_dashboard_task.py]: Updated it.

File: backend/metrics/calculations.py (hunk 3 above)
Comment (talkgarima) [reply backend/metrics/calculations.py]: Updated

File: backend/metrics/calculations.py (hunk 2 above)
Comment (talkgarima) [reply backend/metrics/calculations.py]: updated

File: backend/metrics/calculations.py (hunk 1 above)
Comment (talkgarima) [reply backend/metrics/calculations.py]: Added back

**Review by Nozziel — APPROVED** [review]

**Pull request merged**